src/components/Modals/Onboarding/OnboardingTransfer.vue

```diff
@@ -59,16 +70,30 @@ type TransferClassWithOptionalCourse = {
   course?: CornellCourseRosterCourse;
 };
 
+type TransferExamInfo = {
+  exams: FirestoreTransferExam[];
+  scores?: readonly number[];
+};
```

Comment:
Out of curiosity, what is the benefit of putting `scores` here and using a new type `TransferExamInfo`?

Reply:
Just reduced a bit of code duplication between "examsAP, examsIB" and "scoresAP, scoresIB." Before, we had functions that were very similar for "examsAP" and "examsIB" etc

Reply:
Despite the similarity in naming, `scoresAB` and `scoresIB` are very different from `examsAP` and `examsIB`. They are just global constants -- I don't think they need to be part of a new type. I was actually confused at first because I thought `scores` referred to user scores.

IMO, if we just have `FirestoreTransferExam[]` (like for case exams / if we keep `scoresAB` and `scoresIB` separate), then a new type isn't necessary. Does that make sense?

Reply:
Yeah that makes sense. How does it look now?